### DISCUSSION TITLE
Error during cluster setup

### DISCUSSION THREAD

**Discussion by cedrichelewaut** [microsoft/farmvibes-ai#33] [Q&A] [answered]
Hi, 

I am trying to set up my cluster on Ubuntu and I get a message telling me it is created successfully.
However, I am unable to get the cluster info and everything that follows also fails. I installed all dependencies using the provided file.

Here is my output:
![image](https://user-images.githubusercontent.com/109485561/215256413-8af07565-a2e0-46bb-8b12-1fc3c98f6ad1.png)

**Comment by rafaspadilha** [comment, marked as answer]
Hi, @cedrichelewaut, thank you for using FarmVibes.AI.

Are you still having issues to setup FarmVibes with the current version? 

Sorry for the delay on the reply. In the future, I suggest opening an issue so we can internally track these problems and unblock you quickly.

**Reply by cedrichelewaut** [reply]
I will check it out asap if it works now, will get back to you

**Reply by rafaspadilha** [reply]
Hey, @cedrichelewaut. Were you able to setup FarmVibes.AI? 

I'll close this discussion for now. Feel free to open an issue if the problem persists.
FYI, we just released a new and improved version :)
